### PR TITLE
Add option to invert the color value

### LINKED ISSUE #4333: Add "invert" option to histogram
The min/max sliders are nice, but don't allow to invert the image.

### PR DESCRIPTION
**Pull request by MichaelBuessemeyer**
Thir PR adds the option to invert the color to the dataset layer settings.

Currently, the histogram is not inverted/mirrored when the color is inverted. Shall I also add this @normanrz?

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Open a dataset in view mode, open the dataset settings and toggle the `Invert Color` option on and off. Light areas should turn dark and the other way round.
### Issues:
- fixes #4333

------
- [x] Updated [changelog](../blob/master/CHANGELOG.md#unreleased)
- ~~[ ] Updated [migration guide](../blob/master/MIGRATIONS.md#unreleased) if applicable~~
- ~~[ ] Updated [documentation](../blob/master/docs) if applicable~~
- ~~[ ] Adapted [wk-connect](https://github.com/scalableminds/webknossos-connect) if datastore API changes~~
- ~~[ ] Needs datastore update after deployment~~
- [x] Ready for review

### CODE REVIEW

**Comment by normanrz** [comment]
> Currently, the histogram is not inverted / mirrored when the color is inverted. Shall I also add this @normanrz?

I don't think that is necessary

@youri-k maybe this option needs to be added to the on-disk default configuration?

**Comment by MichaelBuessemeyer** [comment]
Now also solves  #4363 as an own PR would be overkill for this.
Done in commit `use "Volume Layer" as the displayed name for segmentation layer`

File: frontend/javascripts/oxalis/view/settings/dataset_settings_view.js [hunk 1]
```diff
@@ -225,7 +225,7 @@ class DatasetSettings extends React.PureComponent<DatasetSettingsProps, State> {
       <Row style={{ marginTop: isDisabled ? 0 : 16 }}>
         <Col span={24}>
           {this.getEnableDisableLayerSwitch(isDisabled, onChange)}
-          <span style={{ fontWeight: 700 }}>{layerName}</span>
+          <span style={{ fontWeight: 700 }}>{isColorLayer ? layerName : "Volume Layer"}</span>
```
Comment (philippotto) [inline frontend/javascripts/oxalis/view/settings/dataset_settings_view.js]: It should only say "Volume Layer" if it's a volume (or hybrid) tracing :)

File: frontend/javascripts/oxalis/view/settings/dataset_settings_view.js [hunk 2]
```diff
@@ -272,12 +272,28 @@ class DatasetSettings extends React.PureComponent<DatasetSettingsProps, State> {
               }
             />
             {isColorLayer && layer != null ? (
-              <ColorSetting
-                label="Color"
-                value={Utils.rgbToHex(layer.color)}
-                onChange={_.partial(this.props.onChangeLayer, layerName, "color")}
-                className="ant-btn"
-              />
+              <Row className="margin-bottom" align="top">
```
Comment (philippotto) [inline frontend/javascripts/oxalis/view/settings/dataset_settings_view.js]: I'd change the order from:
"Invert Color"  [switch] "Color" [color picker]
to
"Color" [color picker]  "Inverted"  [switch]

That way, the inverted-label doesn't need to contain the word "color".

File: frontend/javascripts/oxalis/view/settings/dataset_settings_view.js (hunk 1 above)
Comment (MichaelBuessemeyer) [reply frontend/javascripts/oxalis/view/settings/dataset_settings_view.js]: Please check my newest changes. Is this what you meant? Now I only use the name "Volume Layer" if the layer is not a color layer and the tracing has a volume.

Comment (philippotto) [reply frontend/javascripts/oxalis/view/settings/dataset_settings_view.js]: Yes, looks good :+1:

**Comment by philippotto** [comment]
Sorry to bring this up again, but the layout still does not convince me :see_no_evil: Right now, the color rectangle and the "invert" label are very close to each other which makes it look like they would belong together (see screenshot). Maybe just split the two UI elements into two rows. That's probably the easiest / clearest.

Current version:
image

**Comment by hotzenklotz** [comment]
How about using an icon to toggle the inversion instead of the checkbox? 

Something like: 
image
(Source: https://ant.design/components/icon/)

**Comment by MichaelBuessemeyer** [comment]
@hotzenklotz The newest version is now using an Icon to invert the layer.

This is how it looks when the layer is not inverted:
image
And this is how it looks like when the color is inverted.
image

Currently, I am not happy with the icon I chose, but I could an icon that fits better for this purpose.
Do you have a better suggestion @hotzenklotz @philippotto?

**Comment by hotzenklotz** [comment]
We still use Fontawesome, right?

Ideas:
https://fontawesome.com/icons/adjust?style=solid
https://fontawesome.com/icons/tint?style=solid

**Comment by MichaelBuessemeyer** [comment]
@hotzenklotz Thanks for your suggestions. I also added some visual feedback for the user to indicate the inversion of the color:

Peek 2020-01-19 22-33

Does this version look fine to you (visually / UX) or do you have other awesome improvements?

**Comment by MichaelBuessemeyer** [comment]
@philippotto Could you please review this PR again? I'd like to merge it soon :slightly_smiling_face:

**Comment by hotzenklotz** [comment]
> Does this version look fine to you (visually / UX) or do you have other awesome improvements?

Nice! I really like the idea with the little animation.

**Pull request merged**